File: AppWeb/API-/app/Models/Reservation.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Reservation extends Model
{
    use HasFactory;

    protected $fillable = ['employeeId', 'table_id', 'dateCreated', 'dateReservation', 'nrPeople', 'comments'];

    public function employee() {
        return $this->belongsTo(Employee::class, 'employeeId');
    }

}

File: AppMov/MyApplication2/app/src/main/java/com/example/myapplication/ui/models/Reservation.kt
package com.example.myapplication.ui.models

import com.google.gson.annotations.SerializedName

data class Reservation (
    @SerializedName("id") val id: Int = 0,
    @SerializedName("employeeId") val employeeId: Int,
    @SerializedName("table_id") val tableNr: Int,
    @SerializedName("dateReservation") val dateReservation: String,
    @SerializedName("nrPeople") val nrPeople: Int,
    @SerializedName("comments") val comments: String = "",
    @SerializedName("created_at") val dateCreated: String? = null

)